gcp_variant_transforms/libs/processed_variant.py

```diff
@@ -599,6 +624,27 @@ def _alt_matches_annotation_alt_minimal_mode(
       return True
     return False
 
+  def _add_annotations_by_allele_num(
+      self, proc_var, annotation_dict, annotation_field_name):
+    # type: (ProcessedVariant, Dict[str, str], str) -> None
+    if _ALLELE_NUM_ANNOTATION not in annotation_dict:
```

Comment:
can we always use this check so that we no longer need the use_allele_num flag? (i.e. the logic would be to first look for allele_num, if it doesn't exist, then use minimal or exact match depending on the --minimal flag). Or are you concerned about cases where allele_num is incorrect?

Reply:
I see. Sounds good. Let's keep the flag then.